src/scripts/migrate.py

```diff
@@ -0,0 +1,189 @@
+"""Slurp migration pipeline
+
+Resources
+- https://incatools.github.io/ontology-access-kit/
+- https://incatools.github.io/ontology-access-kit/intro/tutorial02.html
+
+TODO's:
+  -
+"""
+import sys
+from argparse import ArgumentParser
+from typing import Dict, Iterator, List, Tuple, Union
+
+import curies
+import pandas as pd
+import oaklib
+import yaml
+from oaklib.implementations import ProntoImplementation, SqlImplementation
+from oaklib.resource import OntologyResource
+
+
+# TODO: implement this func
+# todo: IDs should be int or str? prolly str
+def determine_next_available_mondo_id(min_id: str, mondo_termlist_df: pd.DataFrame) -> str:
+    """Starting from `min_id`, count up and check until finding the next ID."""
+    next_id = str(0)
+    return next_id
+
+
+def _get_direct_parents(ontology: Union[ProntoImplementation, SqlImplementation], curie: str):
```

Comment:
@matentzn Having some trouble getting both `ProntoImplementation` and `SqlImplementation` to return relationships (parents or otherwise) of a given term by its URI or CURIE. I created some issues:
- https://github.com/INCATools/ontology-access-kit/issues/237
- https://github.com/INCATools/ontology-access-kit/issues/238

Fortunately, I have a temporary solution that should work until one of these is fixed.